plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py

```diff
@@ -103,11 +103,11 @@ def uiCreate(cls, parentLayout, relativeToWhat):
         cmds.frameLayout(label=kFileOptionsStr, collapsable=False)
         widgetColumn = cmds.columnLayout()
         cmds.checkBox(cls.kMakePathRelativeCheckBox, label=kMakePathRelativeStr, ann=kMakePathRelativeAnnStr)
-        if relativeToWhat == 'SceneFile':
```

Comment:
Did you check with Nat on this? The last time I spoke with her, she was still thinking about the preview fields for layer workflow and hadn't designed them yet.

Reply:
I just ask Natalia on slack and she said it was fine.